core/src/test/java/com/sequenceiq/cloudbreak/structuredevent/rest/urlparsers/RestUrlParserTest.java

```diff
@@ -92,7 +92,7 @@ public class RestUrlParserTest {
             "cluster_templates", "/v4/events", "/v4/diagnostics", "/v4/progress", "/v4/operation",
             "/v4/custom_configurations"};
 
-    private final String[] excludePaths = {"/stacks/internal/crn"};
+    private final String[] excludePaths = {"/stacks/internal/crn", "/govCloud"};
```

Comment:
Nit (and unrelated to the present changes): `excludes` and `excludePaths` should rather by constants, i.e. `private static final String[] EXCLUDES` and `EXCLUDE_PATHS`.